ixnetwork_open_traffic_generator/trafficitem.py

```diff
@@ -215,12 +215,13 @@ def _configure_tracking(self, flow, ixn_tracking):
         if set(tracking_options) != set(ixn_tracking.TrackBy):
             ixn_tracking.update(TrackBy=tracking_options)
         
-    def _configure_options(self, flow):
+    def _configure_options(self):
         enable_min_frame_size = False
-        if (len(flow.packet) == 1 and flow.packet[0].choice == 'pfcpause'
-                and flow.size is not None
-                and flow.size.choice == 'fixed' and flow.size.fixed <= 64):
-            enable_min_frame_size = True
+        for flow in self._api.config.flows:
+            if (len(flow.packet) == 1 and flow.packet[
```

Comment:
This will not address the scenario where there are multiple flows, one of which is a pfc pause flow (what the customer is seeing).
It should be if there is a /globals/apperror/errors that is 'Frame Size Adjusted' and a pfc pause flow then set the min frame size regardless of other flows.

Reply:
@ajbalogh, could you please clarify why you think it won't work for multiple flows ? These changes were done to exclusively take into account multiple flows (with at least one pfc pause flow).

@Rangababu-R could you please confirm if following test worked as expected after changing frame size in pause flow from 128B to 64B (I think you already did, just need a confirmation) ?
- https://github.com/open-traffic-generator/ixnetwork/blob/master/tests/pfc/test_pfc_pause_e2e.py